File: src/main/java/com/github/eendroroy/sdk/openweather/config/DefaultWeatherConfiguration.java
package com.github.eendroroy.sdk.openweather.config;

/**
 * @author indrajit
 */

public class DefaultWeatherConfiguration implements WeatherConfiguration {
    @Override
    public String getBaseUrl() {
        return "https://samples.openweathermap.org/";
    }

    @Override
    public String getAppId() {
        return "439d4b804bc8187953eb36d2a8c26a02";
    }
}

File: src/main/kotlin/com/github/eendroroy/sdk/openweathermap/config/DefaultWeatherConfiguration.kt
package com.github.eendroroy.sdk.openweathermap.config

/**
 * @author indrajit
 */
class DefaultWeatherConfiguration : WeatherConfiguration {
    override fun baseUrl(): String {
        return "https://samples.openweathermap.org/"
    }

    override fun appId(): String {
        return "439d4b804bc8187953eb36d2a8c26a02"
    }
}